src/stories/utils/with-context.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+
+  import { setContext, type Snippet } from "svelte";
+
+  interface Props {
+    children?: Snippet;
+    context: Record<string, any>;
```

Comment:
should context be an optional prop? we're defaulting it to an empty object below and some stories (`src/stories/state-needs-auth-info.stories.svelte`) aren't passing context, just to make things type-safe

Reply:
Done